### ISSUE TITLE
`Cannot read property 'removeChild' of undefined` error

### ISSUE THREAD

**Issue by ehmicky** [Munter/netlify-plugin-subfont#32]
The following error is happening in some builds:

```
Running subfont version 4.2.2
...

pluginInternal Cannot read property 'removeChild' of undefined 
    /opt/build/repo/node_modules/assetgraph/lib/relations/CssImport.js:71:21 CssImport.detach
    /opt/build/repo/node_modules/subfont/lib/subsetFonts.js:270:20 asyncLoadStyleRelationWithFallback
    /opt/build/repo/node_modules/subfont/lib/subsetFonts.js:1333:11 subsetFonts
```

The build logs before this error message are normal and do not indicate anything more.
Do you know where this might come from?

**Comment by Munter** [comment]
That looks like a reference that should have contained a Dom node is actually undefined. Some vector we haven't accounted for in subfont.

Is there an example page that reproduces this?

**Comment by ehmicky** [comment]
Unfortunately not. This is a private repository.

**Comment by ehmicky** [comment]
This happened again, in another private repository, with a little more information:

```
Failed during font subsetting

### LOG: ###
verions: {
  "netlify-build": "1.0.8",
  "netlify-plugin-subfont": "5.0.2",
  "subfont": "5.0.1",
  "assetgraph": "6.0.7"
}

Plugin configuration: {
  "entryPoints": [
    "*.html"
  ],
  "recursive": true,
  "inlineFonts": false,
  "inlineCss": false,
  "fontDisplay": "swap",
  "fallbacks": true
}

Resolved entry points: [
  "/opt/build/repo/dist/404.html",
  "/opt/build/repo/dist/index.html"
]

Subfont called with: {
  "recursive": true,
  "inlineFonts": false,
  "inlineCss": false,
  "fontDisplay": "swap",
  "fallbacks": true,
  "inputFiles": [
    "/opt/build/repo/dist/404.html",
    "/opt/build/repo/dist/index.html"
  ],
  "root": "dist",
  "canonicalRoot": "https://priceless-franklin-02193b.netlify.app",
  "inPlace": true
}

Cannot read property 'removeChild' of undefined 
    /opt/buildhome/.netlify-build-plugins/node_modules/assetgraph/lib/relations/CssImport.js:71:21 CssImport.detach
    /opt/buildhome/.netlify-build-plugins/node_modules/subfont/lib/subsetFonts.js:272:20 asyncLoadStyleRelationWithFallback
    /opt/buildhome/.netlify-build-plugins/node_modules/subfont/lib/subsetFonts.js:1357:11 subsetFonts
    internal/process/task_queues.js:62:5 runNextTicks
    internal/timers.js:429:9 processImmediate
    /opt/buildhome/.netlify-build-plugins/node_modules/subfont/lib/subfont.js:150:24 async subfont
    /opt/buildhome/.netlify-build-plugins/node_modules/netlify-plugin-subfont/lib/index.js:69:5 async onPostBuild
    src/plugins/child/run.js:18:3 async Object.run
    src/plugins/child/main.js:38:38 async handleEvent
    src/plugins/ipc.js:97:9 async process.<anonymous>
```

**Comment by Munter** [comment]
Released in v5.0.3

**Comment by ehmicky** [comment]
Great! :tada: 

Would it be possible for you to bump the version of this plugin in [`netlify/plugins`](https://github.com/netlify/plugins/pulls)? That's the version that is used when users installed plugins in the Netlify app.
Thanks!
